trial/src/Trial.hs

```diff
@@ -84,14 +107,14 @@ data Fatality
     | E
     deriving stock (Show, Eq, Enum, Bounded)
 
-{- |
+{- | 'Warning' pattern synonym.
```

Comment:
Maybe we can add how to use it in pattern matching (for `Warning` and `Error` together)?